packages/core/src/components/Divider/index.tsx

```diff
@@ -7,28 +7,34 @@ export type Props = {
   bottom?: SpacingRange;
   /** Spacing on the top. */
   top?: SpacingRange;
+  /** Render the divider with a short width. */
+  short?: boolean;
 };
 
 /** A horizontal divider. */
 export class Divider extends React.Component<Props & WithStylesProps> {
   static defaultProps: Props = {
     bottom: 2,
     top: 2,
+    short: false,
   };
 
   render() {
-    const { cx, styles, bottom, top } = this.props;
+    const { cx, styles, bottom, short, top } = this.props;
 
     return (
       <Spacing bottom={bottom} top={top}>
-        <div className={cx(styles.divider)} />
+        <div className={cx(styles.divider, short && styles.divider_short)} />
```

Comment:
can you add a test to test that the className has changed when short is passed in ? an example would be, with `short` passed on the wrapper component:

```
expect(wrapper.prop('className')).toMatch('divider_short');
```